package.json

```diff
@@ -7,19 +7,19 @@
   "repository": "gulpjs/eslint-config-gulp",
   "license": "MIT",
   "engines": {
-    "node": ">= 0.10"
+    "node": ">=10"
```

Comment:
I'll remove the space between >= and 10 in the boilerplate too when updating to 10.12 👍